path_config_test.go

```diff
@@ -499,6 +499,7 @@ func ldapResponseData(vals ...interface{}) map[string]interface{} {
 		"case_sensitive_names":   false,
 		"certificate":            "",
 		"deny_null_bind":         true,
+		"dereference_aliases":    "never",
```

Comment:
One thing about LDAP secrets is that it shares library code for configuration with LDAP auth. It returns all config fields, some of which are only relevant to LDAP auth. `dereference_aliases` was added to the library code recently in https://github.com/hashicorp/vault/pull/18230. I believe this is showing up because you're updating the Vault SDK which pulls in the new config parameter.

It would be ideal if we filtered LDAP auth config fields that aren't relevant to LDAP secrets, but that ship has sailed. So I think this is okay @averche.

Reply:
Thanks for the explanation!